packages/react-vapor/src/components/tooltip/DateTooltip.tsx

```diff
@@ -0,0 +1,24 @@
+import * as moment from 'moment';
+import * as React from 'react';
+import * as _ from 'underscore';
+import {Tooltip} from './Tooltip';
+
+export type DateTooltips = moment.Moment | string;
+
+export interface DateTooltipsProps {
+    date: DateTooltips;
+    format?: ((date: moment.Moment) => string) | string;
+    tooltipFormat?: string;
+}
+
+export const DateTooltip: React.FunctionComponent<DateTooltipsProps> = ({date, format, tooltipFormat}) => {
+    let content: string;
+    const title = tooltipFormat ? moment(date).format(tooltipFormat) : moment(date).format('LLL');
+    const momentDate = _.isString(date) ? moment(date) : date;
```

Comment:
If `date` is a string, `momentDate` will be a string too right?

In the next `if-else` block, it will throw an error when calling `.format` on a string.
![image](https://user-images.githubusercontent.com/35579930/71179768-ce23f080-223e-11ea-87bd-679b5f7647ad.png)

There should be a unit test for this case.

Reply:
IIRC moment(date) will change string type to object type. In this case, moment object